hdbt.theme

```diff
@@ -635,10 +636,12 @@ function hdbt_preprocess_block(&$variables): void {
 
         if (!empty($params)) {
           $entity_type = key($params);
+          /** @var \Drupal\node\NodeInterface $node */
           $node = Drupal::entityTypeManager()
             ->getStorage($entity_type)
-            ->load($params[$entity_type])
-            ->getTranslation($variables['current_langcode']);
+            ->load($params[$entity_type]);
+
+          $node = $node->getTranslation($variables['current_langcode']);
```

Comment:
This piece of code should have always checked if the translation exists

`$node = $node->hasTranslation($variables['current_langcode']) ? $node->getTranslation($variables['current_langcode']) : $node`